messaging/mail_sender_classes.py

```diff
@@ -114,7 +114,13 @@ class ConnectedUserMailSender(MasterMailSender):
     """
     def get_real_receivers_list(self):
         if self.connected_user:
```

Comment:
if self.connected_user and self.connected_user.person.email:
    return [...]
else
   logger.error(...)